seqr/views/utils/orm_to_json_utils.py

```diff
@@ -693,11 +697,32 @@ def get_project_collaborators_by_username(project, include_permissions=True):
             collaborator, include_permissions, can_edit=True
         )
 
+    if project_has_anvil(project):
+        permission_levels = get_workspace_collaborator_perms(user, project.workspace_namespace, project.workspace_name)
+        for email, permission in permission_levels.items():
+            collaborator = User.objects.filter(email=email)
```

Comment:
this is really inefficient in terms of database queries, which can be slow. Querying 100 users takes roughly the same time as querying 1 so I feel really strongly that you need to do one query here. Something like 
```
users_by_email = {u.email: u for u in User.objects.filter(email__in = permission_levels.keys())}
for email, permission in permission_levels.items():
    collaborator = users_by_email.get(email)
    if collaborator:
    ...
```

Reply:
Okay.